android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java

```diff
@@ -443,6 +443,8 @@ private void selectSpeakerAsAudioOutput() {
     private void connectToRoom(String roomName, String accessToken) {
         audioSwitch = new AudioSwitch(getApplicationContext());
 
+        selectSpeakerAsAudioOutput();
```

Comment:
why this? what if the user is on headphones?

Reply:
now, there is a condition to check if its earpiece